installer.sh

```diff
@@ -15,37 +15,80 @@ MSWEBVIEW_VERSION=1.0.3351.48
 MSWEBVIEW_INCLUDE_DIR=$PACKAGE_DIR/Microsoft.Web.WebView2.$MSWEBVIEW_VERSION/build/native/include
 MSWEBVIEW_LIB_DIR=$PACKAGE_DIR/Microsoft.Web.WebView2.$MSWEBVIEW_VERSION/build/native/x64
 
+
+
+CHROMIUM_VERSION=138.0.17
+CHROMIUM_INCLUDE_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_VERSION/build/native/include
+CHROMIUM_LIB_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_VERSION/build/native/x64
+
 SYS_OPTS=""
 
 PATH=$(cygpath -w /ucrt64/bin):$PATH
 export PATH
-
 source ./.scripts/util.sh
 # For Windows, check that we are running in a MSYS2 UCRT environment
 check_env
 
 PACKAGE_INSTALL_HELP="\
 #  Installs required library packages.
-#  ATM only MSWEBVIEW2 on Windows is supported, so we leave this as basic logic.
-#  @todo Expand the logic for mutiple OS and embedded browser packages.
+#  ATM only MSWEBVIEW2 on Windows is fully supported, so we leave this as basic logic.
+#  The installer currently supports CEF 
+#  @todo Expand the logic for multiple OS and embedded browser packages.
 ## Usage:
 #   packages_install\
 "
+
```

Comment:
This is the user entry function, so should be named `select_browser(){...}` as per the corresponding documentation.